cmd/root.go

```diff
@@ -128,6 +128,28 @@ func namespace() (string, error) {
 	return ns, err
 }
 
+// server returns the server URI and namespace of the cluster at the provided
+// context. If context is nil, the current context is used.
+func server(context *string) (uri, namespace string, err error) {
```

Comment:
One last thing. `server` might not be the best name for this. Really you're resolving context to a URI and a namespace, right? Maybe `resolveContext`?